packages/core/src/format/format.ts

```diff
@@ -134,6 +134,24 @@ export function formatProblems(
       }
       break;
     }
+    case 'markdown': {
+      const groupedByFile = groupByFiles(problems);
+      for (const [file, { fileProblems }] of Object.entries(
+        groupedByFile
+      )) {
+        logger.info(`# Lint: ${isAbsoluteUrl(file) ? file : path.relative(cwd, file)}\n\n`);
```

Comment:
I'd change the formatting a bit, like this:
```suggestion
        logger.info(`**${isAbsoluteUrl(file) ? file : path.relative(cwd, file)}**\n\n`);
```

Otherwise looks quite nice. Thanks!

Reply:
I went for making it into a subheading instead so it will still be a title, but will work if linting multiple files